tests/integration/UploadSlideBackTranslationIntegrationTest.php

```diff
@@ -0,0 +1,317 @@
+use Symfony\Component\Process\Process;
+use PDO;
+
+class UploadSlideBackTranslationIntegrationTest extends TestCase
+{
+    private static $db;
+
+    /** @var Process */
+    private static Process $process;
+
+    private static string $host = "localhost:8899";
+
+    private string $storyTemplate = 'test_story';
+    private string $storyTemplate2 = 'test_story2';
+
+    private string $uri = '/API/uploadSlideBacktranslation.php';
+
+    private static string $filesRoot;
+    private static string $uploadedProjectDir;
+    private static Filesystem $fileSystem;
+
+    private Client $httpClient;
+
+    public static function setUpBeforeClass(): void
+    {
+        // start php in-built server
+        self::$process = new Process(["php", "-S", self::$host, "-t", ROOT_PATH . "."]);
+        self::$process->start();
+        usleep(100000);
+
+        // create required directory for test data
+        self::$fileSystem = new Filesystem();
+        self::$filesRoot = $GLOBALS['filesRoot'];
+        $templateDir = self::$filesRoot . "/Templates";
+        self::$fileSystem->mkdir($templateDir);
+        // copy test story template files to files root
+        self::$fileSystem->mirror(dirname(__DIR__) . "/data/templates", $templateDir);
+        self::$uploadedProjectDir = self::$filesRoot . "/Projects";
+
+        self::$db = new PDO(DB_DNS);
+        self::$db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+    }
+
+    public static function tearDownAfterClass(): void
+    {
+        self::$fileSystem->remove(self::$filesRoot);
+        self::$process->stop();
+        self::$db->query('DELETE FROM Slide');
+        self::$db->query('DELETE FROM Stories');
+        self::$db = null;
+    }
+
+
+    public function setUp(): void
+    {
+        $this->httpClient = new Client(['http_errors' => false]);
+    }
+
+    /**
+     *
+     * @return int
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     * @throws \Exception
+     */
+    public function testCreateNewStory(): void
+    {
+        // request payload
+        $payload = [
+            'Key' => '',
+            'PhoneId' => 'rstuvw',
+            'Data' => base64_encode('Test data for story file'),
+            'TemplateTitle' => $this->storyTemplate,
+            'IsWholeStory' => "true"
+        ];
+
+        $response = $this->httpClient->request("POST", self::$host . $this->uri, ['form_params' => $payload]);
+        $responseJson = $response->getBody()->getContents();
+
+        $this->assertEquals(200, $response->getStatusCode());
+        $responseArr = json_decode($responseJson, true);
+        $this->assertArrayHasKey('StoryId', $responseArr);
+        $storyId = $responseArr['StoryId'];
+
+
+        # verify story is created correctly in the database
+        $stories = $this->getStories($storyId);
+        $this->assertCount(1, $stories);
+
+        $this->assertEquals([
+            "id" => $storyId,
+            "title" => "test_story",
```

Comment:
Looks like only the title and projectId are interesting. Can you only assert those two?